glue_ginga/plugins/Glue.py

```diff
@@ -309,8 +309,8 @@ def start_glue_cb(self):
 
         # self.glue_app._create_terminal()
         sgeo = self.glue_app.app.desktop().screenGeometry()
-        self.glue_app.resize(sgeo.width() * 0.9, sgeo.height() * 0.9)
         self.glue_app.show()
```

Comment:
NOTE: Using `show()` here instead of `start()` caused IPython terminal to be unavailable in Glue launch from the plugin. See https://github.com/ejeschke/glue-ginga/issues/22#issuecomment-325498218

Reply:
And just to be clear, what happens if you do ``start``?

Reply:
After Glue is started from within the plugin, Ginga cannot load new images properly anymore. Sometimes, the buffer loads on main display (in Ginga) but thumbnail does not show, sometimes nothing loads at all. Basically, Ginga stops working.

Reply:
> Basically, Ginga stops working.

We need to start glue without committing to it's Qt event loop.  The Ginga event loop will make calls back into glue to handle any QT events.